perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -1617,6 +1617,20 @@ public boolean haveStatus(PerunSession sess, Member member, Status status) {
 		return member.getStatus().equals(status);
 	}
 
+	@Override
+	public boolean isUserValidMember(PerunSession sess, User user, Vo vo) {
+		// check if given user is VALID member of VO
```

Comment:
Looks good, just cosmetic suggestion - I'd say both comments are not necessary and can be removed.